README.md

```diff
@@ -309,8 +296,8 @@ branch. PRs should be based on the `pyicloud-ipd` branch and submitted to
 
 ### Building the Docker image
```

Comment:
This section is duplicate of what is available above. Probably worth aggregating.

Reply:
I thought the same but did not want to rearrange the thing without discussion. Maybe it makes sense to put this somewhere separate as well in the future. We also have some FAQs already, p.e. problems on first run, how to handle directory-structure, etc. I thought this could be worth a wiki-page where others also might contribute, what do you think? Could be linked just here and updated outside releases.